build-scripts/ubuntu-1604/postinst_node

```diff
@@ -120,6 +120,57 @@ TimeoutSec=300
 WantedBy=multi-user.target
 EOF
 
+# add supervisord script
+cat <<EOF > /etc/supervisor/indy-node.conf
+[supervisord]
+nodaemon=true
+logfile=/var/log/indy/supervisord.log
+logfile_maxbytes=10MB
+loglevel=critical
+
+[supervisorctl]
+serverurl=http://127.0.0.1:9001
+
+[inet_http_server]
+port = 127.0.0.1:9001
+
+[rpcinterface:supervisor]
+supervisor.rpcinterface_factory = supervisor.rpcinterface:make_main_rpcinterface
+
+[program:ntp]
+command=/usr/sbin/ntpd -n
+autorestart=true
+startsecs=15
+startretries=6
+stopasgroup=true
+killasgroup=true
+
+[program:indy-node]
+command=/usr/local/bin/init_and_start_indy_node.sh
```

Comment:
Why do you need to couple `init_indy_node` and `start_indy_node`? As it is [documented](https://github.com/hyperledger/indy-node/blob/master/docs/start-nodes.md#scripts-for-initialization) for now keys initialization is a separate operation and shouldn't be performed each time node starts.